src/main/java/org/squiddev/plethora/gameplay/modules/glasses/objects/object3d/Line3D.java

```diff
@@ -0,0 +1,144 @@
+package org.squiddev.plethora.gameplay.modules.glasses.objects.object3d;
+
+import com.google.common.base.Objects;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.client.renderer.BufferBuilder;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.Tessellator;
+import net.minecraft.client.renderer.vertex.DefaultVertexFormats;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.Vec3d;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+import org.lwjgl.opengl.GL11;
+import org.squiddev.plethora.api.method.MethodResult;
+import org.squiddev.plethora.api.method.wrapper.FromTarget;
+import org.squiddev.plethora.api.method.wrapper.PlethoraMethod;
+import org.squiddev.plethora.gameplay.modules.glasses.CanvasClient;
+import org.squiddev.plethora.gameplay.modules.glasses.objects.ColourableObject;
+import org.squiddev.plethora.gameplay.modules.glasses.objects.ObjectRegistry;
+import org.squiddev.plethora.gameplay.modules.glasses.objects.Scalable;
+import org.squiddev.plethora.utils.ByteBufUtils;
+
+import javax.annotation.Nonnull;
+
+public class Line3D extends ColourableObject implements Positionable3D, DepthTestable, Scalable {
```

Comment:
OK, it feels really iffy having a line be "set position" + "set end position". I'll be the first to admit that the multi-line interface isn't the friendliest thing to use in Lua (or Java for that matter), but this feels ... ewww.